src/Kovan-DssSpell.sol

```diff
@@ -74,6 +72,7 @@ contract SpellAction {
     // A table of rates can be found at
     //    https://ipfs.io/ipfs/QmefQMseb3AiTapiAKKexdKHig8wroKuZbmLtPLv4u2YwW
     //
+    uint256 constant ONE_PERCENT_RATE   = 1000000000315522921573372069;
     uint256 constant THREE_PERCENT_RATE = 1000000000937303470807876289;
```

Comment:
Looks like this one isn't used and can be removed.

Reply:
removed